src/citrine/builders/predictors.py

```diff
@@ -134,3 +144,112 @@ def _build_mean_property_predictor(
         impute_properties=impute_properties,
         label=label
     )
+
+
+# Add simple ml builder
+def build_simple_ml(
+    *,
+    project: Project,
+    name: str,
+    description: str,
+    inputs: List[Descriptor],
+    outputs: List[Descriptor],
+    latent_variables: List[Descriptor],
+    training_data: Optional[List[DataSource]] = None
+) -> GraphPredictor:
+    """Build a graph predictor that connects all inputs to all latent variables and outputs.
+
+    Parameters
+    ----------
+    project: Project
+        Project that contains the predictor to be built
+    name: str
+        name of the returned predictor
+    description: str
+        the description of the predictor
+    inputs: list[Descriptor]
+        Descriptors that represent inputs to relations
+    outputs: list[Descriptor]
+        Descriptors that represent outputs of relations
+    latent_variables: list[Descriptor]
+        Descriptors that are predicted from inputs and used when predicting the outputs
+    training_data: Optional[List[DataSource]]
+        Sources of training data. Each can be either a CSV or an GEM Table. Candidates from
+        multiple data sources will be combined into a flattened list and de-duplicated by uid and
+        identifiers. de-duplication is performed if a uid or identifier is shared between two or
+        more rows. The content of a de-duplicated row will contain the union of data across all
+        rows that share the same uid or at least 1 identifier. Training data is unnecessary if the
+        predictor is part of a graph that includes all training data required by this predictor.
+
+    Returns
+    -------
+    GraphPredictor
+        GraphPredictor connecting all inputs to all latent variables and all inputs and latent
+        variables to all outputs.
+    """
+    ml_model_feature_descriptors = [d for d in inputs
+                                    if isinstance(d, (RealDescriptor, CategoricalDescriptor))]
+    chemical_formula_descriptors = [d for d in inputs
+                                    if isinstance(d, (ChemicalFormulaDescriptor))]
+    molecular_structure_descriptors = [d for d in inputs
+                                       if isinstance(d, (MolecularStructureDescriptor))]
```

Comment:
Nitpick: Here and in line 193, since there's only one class, the parentheses around `MolecularStructureDescriptor` and `ChemicalFormulaDescriptor` are unnecessary.